package/firmware/ipq-wifi/Makefile

```diff
@@ -26,6 +26,7 @@ endef
 ALLWIFIBOARDS:= \
 	aruba_ap-303 \
 	avm_fritzrepeater-1200 \
+	ezviz_cs-w3-wd1200g-eup \
```

Comment:
I can't find the board-files in the ath10k mailing-list archive.

Please see the comment in ipq-wifi's Makefile if you need a pointer/template:

    # This is intended to be used on an interim basis until device-specific
    # board data for new devices is available through the upstream compilation
    #
    # Please send a mail with your device-specific board files upstream.
    # You can find instructions and examples on the linux-wireless wiki:
    # <https://wireless.wiki.kernel.org/en/users/drivers/ath10k/boardfiles>

Reply:
I was under the impression that it would take some time for the boardfiles to be included in OpenWRT, so I was planning to submit them after the code was merged to master. Anyway, I now submitted the request to add the board-files (http://lists.infradead.org/pipermail/ath10k/2020-January/014620.html)